deploy/job-manager/deploy.sh

```diff
@@ -28,7 +28,7 @@ function configure_kubernetes() {
     stdout "Setting to use Google project: project ${GCLOUD_PROJECT}"
     gcloud config set project ${GCLOUD_PROJECT}
 
-    stdout "Setting to use GKE cluster: project gke_${GCLOUD_PROJECT}_us-central1-b_listener"
+    stdout "Setting to use GKE cluster: project gke_${GCLOUD_PROJECT}_us-central1-b_lira"
```

Comment:
Now the cluster name is `gke_${GCLOUD_PROJECT}_us-central1-a_green-100-us-central1` right?

Reply:
The Job Manager is deployed to the same project as CAAS, not the mint project. So it will have a different K8s name. I don't think that the name is settled yet, but would likely be "gke_${GCLOUD_PROJECT}_us-central1-a_caas-100-us-central1".

Reply:
Ok, that makes sense!